src/caffe/util/im2col.cu

```diff
@@ -230,33 +255,27 @@ __global__ void col2im_gpu_kernel(const int n, const Dtype* data_col,
     const int w_im = index % width + pad_w;
     const int h_im = (index / width) % height + pad_h;
     const int c_im = index / (width * height);
+    int kernel_extent_w = (kernel_w - 1) * dilation_w + 1;
+    int kernel_extent_h = (kernel_h - 1) * dilation_h + 1;
```

Comment:
Don't worry about this here, because the existing code doesn't follow this convention, but I'll just comment for the future that to me the `h` computations should come before the `w` ones, following argument order and loop order.